src/commands/bundle.zsh

```diff
@@ -9,7 +9,7 @@ antigen-bundle () {
     return 1
   fi
 
-  builtin typeset -A bundle; -antigen-parse-args 'bundle' ${=@}
+  builtin typeset -A bundle; -antigen-parse-args 'bundle' $@
```

Comment:
I'm not super familiar with how themes work. Do you have a test case that fails I could debug?